.github/workflows/test.yml

```diff
@@ -85,8 +85,8 @@ jobs:
       uses: actions/cache@v3
       with:
         path: |
-          weights/whisper-tiny.en.pt
-          weights/gpt2.tiktoken
+          ~/.cache/tinygrad/downloads/3a2cae277fcec4464349df535729caaf
```

Comment:
I don't like this with the hashes, it's hard to read. The whole downloads folder should probably be cached